db/dcrsqlite/blocksretrieval_test.go

```diff
@@ -48,3 +48,45 @@ func TestEmptyDBRetrieveBlockFeeInfo(t *testing.T) {
 			cmp.Diff(*result, defaultChartsData))
 	}
 }
+
+func TestEmptyDBGetBestBlockHash(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := ObtainReusableEmptyDB()
+	str := db.GetBestBlockHash()
+	if str != "" {
+		// Open question: Should it really be the empty string?
+		// Maybe error instead to avoid confusion?
```

Comment:
Certainly an error would convey more information about why it didn't get a block hash, but having `""` as a sentry is fine.  Let's leave the function alone for now.  Any valid hash will have 64 characters (32 bytes hexadecimal encoded).